test/integration/targets/ecs_cluster/meta/main.yml

```diff
@@ -0,0 +1,3 @@
+dependencies:
+  - prepare_tests
```

Comment:
I think prepare_tests is empty

Reply:
Lazy copy and paste on my part. setup_ec2 isn't needed either unless you then include setup_ec2/tasks/common.yml, which most targets don't do.